packages/generator/src/generate.ts

```diff
@@ -65,24 +45,26 @@ const generate = async (props: GenerateProps) => {
       drawEmbeddedPage({ page, embeddedPage, embedPdfBox });
       for (let l = 0; l < keys.length; l += 1) {
         const key = keys[l];
-        const schema = schemas[j];
+        const schema = template.schemas[j];
         const templateSchema = schema[key];
         const input = inputObj[key];
-        const fontSetting = { font, pdfFontObj, fallbackFontName };
-
-        await drawInputByTemplateSchema({
-          input,
-          templateSchema,
-          pdfDoc,
-          page,
-          fontSetting,
-          inputImageCache,
-        });
+
+        if (!templateSchema || !input) {
+          continue;
+        }
+
+        // TODO ↓ If a Custom Schema is defined, it can undergo custom rendering processes by merging it with the renderer object.
+        // const renderer = Object.assign(buildInRenderer, options.schema);
+        const rendererObj = buildInRenderer[templateSchema.type];
+        if (!rendererObj) {
+          throw new Error(`Renderer for type ${templateSchema.type} not found`);
+        }
+        await rendererObj.renderer({ input, templateSchema, pdfDoc, page, options, _cache });
```

Comment:
would `render()` be a better function name (a verb) than `renderer` which is the a noun for the object type?